metadata-ingestion/src/datahub/ingestion/source/ldap.py

```diff
@@ -387,8 +392,10 @@ def build_corp_user_mce(
 
         manager_urn = f"urn:li:corpuser:{manager_ldap}" if manager_ldap else None
 
+        user_urn = email if self.config.use_email_as_username else ldap_user
+
         user_snapshot = CorpUserSnapshotClass(
-            urn=f"urn:li:corpuser:{ldap_user}",
+            urn=f"urn:li:corpuser:{user_urn}",
```

Comment:
cleanup: can we refactor this to use `make_user_urn` (and same for the group urn below)?

Reply:
updated the code.